Bellhop.d.ts

```diff
@@ -0,0 +1,35 @@
+export as namespace BellhopIframe;
+
+interface PriorityFunction extends Function {
+  private _priority: number;
```

Comment:
Interfaces usually don't allow member scoping - I'd be concerned that the compiler would complain about this when a project tries to build against it.